cli/src/main/java/com/devonfw/tools/ide/tool/ide/IdeToolCommandlet.java

```diff
@@ -18,6 +19,8 @@
  */
 public abstract class IdeToolCommandlet extends PluginBasedCommandlet {
 
+  public final FlagProperty skipWorkspaceUpdate;
```

Comment:
Please add Javadoc.

Reply:
Since all that code was reverted and is gone, I will set this to resolved so we can proceed.